helm/flowforge/README.md

```diff
@@ -17,12 +17,13 @@ This chart uses the Bitnami PostgreSQL Chart to provide an instance of a Postgre
  - `forge.registry` the container registry to find Project templates (default Docker Hub)
  - `forge.dbUsername` (default `forge`)
  - `forge.dbPassword` (default `Zai1Wied`)
+ - `forge.dbName` (default `flowforge`)
```

Comment:
This is missing from the docs: https://flowforge.com/docs/install/configuration/#database-configuration

Why is it configurable in the helm charts, but no-where else?

Reply:
https://github.com/flowforge/flowforge/issues/1044